cmake/install_helpers.cmake

```
@@ -5,6 +5,9 @@ include(GNUInstallDirs)
 set(GINKGO_INSTALL_INCLUDE_DIR "${CMAKE_INSTALL_INCLUDEDIR}")
 set(GINKGO_INSTALL_LIBRARY_DIR "${CMAKE_INSTALL_LIBDIR}")
 set(GINKGO_INSTALL_RUNTIME_DIR "${CMAKE_INSTALL_BINDIR}")
+set(GINKGO_INSTALL_FULL_INCLUDE_DIR "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
+set(GINKGO_INSTALL_FULL_RUNTIME_DIR "${CMAKE_INSTALL_FULL_BINDIR}")
+set(GINKGO_INSTALL_FULL_LIB_DIR "${CMAKE_INSTALL_FULL_LIBDIR}")
```

```suggestion
set(GINKGO_INSTALL_FULL_LIBRARY_DIR "${CMAKE_INSTALL_FULL_LIBDIR}")
```
or change the name in another file?

---

You are right, I don't know how I missed this.
I tested it on my side with a patch file which added the cmake variable instead of a ginkgo helper.